appinventor/components/src/com/google/appinventor/components/runtime/PasswordTextBox.java

```diff
@@ -7,8 +7,11 @@
 package com.google.appinventor.components.runtime;
 
 import com.google.appinventor.components.annotations.DesignerComponent;
+import com.google.appinventor.components.annotations.DesignerProperty;
```

Comment:
You add the DesignerProperty import here, but do not use it on the PasswordVisible property defined below. Either remove the import or add the annotation on the property. I think it would be reasonable to make the property visible in the designer.